multiselect.go

```diff
@@ -27,6 +27,9 @@ type MultiSelect struct {
 	Default       []string
 	Help          string
 	PageSize      int
+	VimMode       bool
+	FilterMessage string
```

Comment:
I'm curious what the FilterMessage is in the cli you are working on.

I'd like to provide some instructions to the user, indicating what mode their in. My original thinking was that when we are filtering, we can show the filter message along with the current filter value. A default message might be something like `You can type to filter items`. And when the user is in vim-mode, we could provide a message like `Use the arrow keys or j/k to highlight an option`.

Reply:
^ Does mesh well with my other comment of leaving the filter behind when the user presses `esc`. I worry that we will run into overflow issues (which are only really a problem in windows per #101) but I think if we can find a concise way to provide good instructions, we'd be in awesome shape

Reply:
so `FilterMessage` currently is `" " + m.filter` - i.e. its the thing shown on the screen after the input message.  I guess the template could just conditionally output `" " + m.Filter if m.Filter != ""`

Reply:
BTW we clear `m.filter` after the user accepts the input so it is removed from the screen

Reply:
Ah so you're not expecting it to be modified by the user? Should it be a private field then?

Reply:
I figure d a user may want to configure the default - but I'm not too worried if you want to make it private

Reply:
I suppose there’s no harm in extra configuration. Let’s leave it public. Sorry to be so nit picky. I love the feature and want to make sure we think through all of its impact on both developers and end users.

Reply:
No worries! TBH I'd not even considered vim mode before this PR started so its definitely well worth trying to figure out how all these pieces fit together to get the best UX